apiserver/gui.go

```diff
@@ -86,8 +82,8 @@ func guiEndpoints(pattern, dataDir string, ctxt httpContext) []apihttp.Endpoint
 			})
 		}
 	}
-	add("/config.js", (*guiHandler).serveConfig)
-	add("/static/", (*guiHandler).serveStatic)
+	add(pattern+"config.js", (*guiHandler).serveConfig)
```

Comment:
The registration of the patterns was missing the /dashboard/ prefix.

Reply:
Unfortunately the dashboard application requires that the config and static paths be served from the host directly. This worked before so I'll have to see if I can revert this change and update the tests.